pkg/stages/steps_install.go

```diff
@@ -324,3 +334,626 @@ depend() {
 
 	return data
 }
+
+// GetInstallOemCloudConfigs dumps the OEM files to the system from the embedded oem files
+// TODO: Make them first class yip files in code and just dump them into the system?
+// That way they can be set as a normal yip stage maybe? a yip stage that dumps the yip stage lol
+func GetInstallOemCloudConfigs(l types.KairosLogger) error {
+	files, err := embeddedConfigs.ReadDir("cloudconfigs")
+	if err != nil {
+		l.Logger.Error().Err(err).Msg("Failed to read embedded files")
+		return err
+	}
+
+	// Extract each file
+	for _, file := range files {
+		if !file.IsDir() {
+			data, err := embeddedConfigs.ReadFile(filepath.Join("cloudconfigs", file.Name()))
+			if err != nil {
+				l.Logger.Error().Err(err).Str("file", file.Name()).Msg("Failed to read embedded file")
+				continue
+			}
+
+			// check if /system/oem exists and create it if not
+			if _, err = os.Stat("/system/oem"); os.IsNotExist(err) {
+				err = os.MkdirAll("/system/oem", 0755)
+				if err != nil {
+					l.Logger.Error().Err(err).Str("dir", "/system/oem").Msg("Failed to create directory")
+					continue
+				}
+			}
+			outputPath := filepath.Join("/system/oem/", file.Name())
+			err = os.WriteFile(outputPath, data, 0644)
+			if err != nil {
+				fmt.Printf("Failed to write file %s: %v\n", outputPath, err)
+				continue
+			}
+
+			l.Logger.Debug().Str("file", outputPath).Msg("Wrote cloud config")
+		}
+	}
+	return nil
+}
+
+// GetInstallBrandingStage returns the branding stage
+// This stage takes care of creating the default branding files that are used by the system
+// Thinks like interactive install or recoivery welcome text or grubmenu configs
+func GetInstallBrandingStage(_ values.System, _ types.KairosLogger) []schema.Stage {
+	var data []schema.Stage
+
+	data = append(data, []schema.Stage{
+		{
+			Name: "Create branding files",
+			Files: []schema.File{
+				{
+					Path:        "/etc/kairos/branding/grubmenu.cfg",
+					Permissions: 0644,
+					Owner:       0,
+					Group:       0,
+					Content: `
+menuentry "Kairos remote recovery" --id remoterecovery {
```

Comment:
this is better to isolate it as a constant. hard to find it here if someone needs to change it

Reply:
yep tep, will move everything over a constant